feature-libs/product/configurators/common/styles/_configurator-update-message.scss

```diff
@@ -1,47 +1,36 @@
 %cx-configurator-update-message {
-  .wrapper {
-    position: absolute;
-    height: 100%;
-    width: 100%;
-    pointer-events: none;
-    z-index: 99;
-
-    div#cx-configurator-update-message {
-      pointer-events: auto;
-      position: sticky;
-      top: 0;
+  div {
```

Comment:
Nitpick: I'd rather use CSS class, instead of DIV. But I leave it up to your final decision ;)

Reply:
I will probably add a class in the next PR. I found a rendering bug at that I am working in my next PR